tools/run_tests/run_tests.py

```diff
@@ -440,11 +440,15 @@ def dockerfile_dir(self):
   def _get_tox_envs(self, compiler):
     """Returns name of tox environment based on selected compiler."""
     if compiler == 'default':
-      return ('py27', 'py34')
+      return ('py27', 'py34', 'py35', 'py36')
```

Comment:
@nathanielmanistaatgoogle If we are running the Python 3.5 and Python 3.6 tests by default, we need to modify the Dockerfiles. Is there a specific Dockefile that I should modify or all of them?